proto/error.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+option go_package = "github.com/ray-project/kuberay/proto/go_client";
+package proto;
+
+import "google/protobuf/any.proto";
+
+message Error {
+  string error_message = 1;
+  string error_details = 2;
+}
+
+message Status {
+  string error = 1;
```

Comment:
Is the `error` refer to `message Error`?

Reply:
em. I think we can remove `message Error`. Leave `Status` here for all purpose.

Reply:
em. I think we can remove `message Error`. Leave `Status` here for all purpose.